src/services/GithubService.js

```diff
@@ -0,0 +1,16 @@
+import Axios from "axios";
+
+export default {
+    getPullRequests: (username) => {
```

Comment:
@lazyc97 I suggest `getPullRequests` is wrong because this request fetch issues, it should be something like  `getIssuesByUsername` or `getUserIssues`